packages/rlc-common/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.38.5 (2025-03-24)
+
+- [Feature] Improve generateTest config. Please refer to [#3125](https://github.com/Azure/autorest.typescript/pull/3125)
+- [Feature] Support tsconfig changes, vitest config changes and api-extractor changes from migration tool for codegen. Please refer to [#3081](https://github.com/Azure/autorest.typescript/pull/3081)
+- [Feature] Remove Impressions in HLC readme.md and update Api-extractor template for modular. Please refer to [#3092](https://github.com/Azure/autorest.typescript/pull/3092)
+- [Feature] Update smoke test specs to remove deprecated typespec usage. Please refer to [#3103](https://github.com/Azure/autorest.typescript/pull/3103)
+- [Feature] To not expose internal operations for hierarchy client true. Please refer to [#3082](https://github.com/Azure/autorest.typescript/pull/3082)
+- [Feature] Upgrade dev dependency typescript version to `~5.7.2~. Please refer to [#3100](https://github.com/Azure/autorest.typescript/pull/3100)
+- [Feature] upgrade to compiler 0.67.x. Please refer to [#3071](https://github.com/Azure/autorest.typescript/pull/3071)
+- [Feature] Update format script. Please refer to [#3094](https://github.com/Azure/autorest.typescript/pull/3094)
+- [Feature] Remove the modular normalization and directly use the rlc-common ones. Please refer to [#3115](https://github.com/Azure/autorest.typescript/pull/3115)
```

Comment:
Can you add one Feature for all name normalization related and list the details there ? https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/CHANGELOG.md#0310-2024-07-23